README.md

```diff
@@ -29,6 +29,7 @@ Singer taps function in two modes: [discovery mode](https://github.com/singer-io
     ```json
     {
         "api_key": "pk_XYZ",
+        "api_version": "2024-02-15",
```

Comment:
Remove api_version from config.json as api_version should not be the configurable property unless your code is flexible enough to switch between different api versions.

Reply:
Sure, I'll make that change.